### DISCUSSION TITLE
`UrlHelper.Content` overload that accepts an `appendVersion` argument

### DISCUSSION THREAD

**Discussion by tuespetre** [dotnet/aspnetcore#58273] [Ideas]
Already submitted this as [an issue](https://github.com/dotnet/aspnetcore/issues/57964) before I noticed the 'Discussions' tab existed, but it quickly became buried under `pr-benchmarks` and other things, lol. So I'm hoping this is the right place to post ideas instead.

**Comment by martincostello** [comment]
_API proposal_ issues are the usual way to suggest new additions to the API surface. They have a specific template and labels and the team triage them regularly: https://github.com/dotnet/aspnetcore/issues/new/choose

**Reply by tuespetre** [reply]
I see... I had assumed using the "feature request" issue would achieve a similar result.
